### PR TITLE
feature/improving_css-GDEV-457

### PR DESCRIPTION
**Pull request by ac-jorellanaf**
1. Added separate scrollbars for datasets and filters.
2. Fixed buttons not being in same div as facets.
3. Fixed total datasets badge position.
4. Fixed facet text and count not being in same line,

### CODE REVIEW

File: src/components/browse/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -5,43 +5,56 @@ import { Row, Col, Button } from "react-bootstrap";
 import { facetModel, facetFilterModel } from "../../../models/facets";
 import { searchResponseModel } from "../../../models/dataset";
 import { getDatasetsSearchResp } from "../../../api/browse";
+import PerfectScrollbar from 'react-perfect-scrollbar'
+import 'react-perfect-scrollbar/dist/css/styles.css';
```
Comment (alamgirmani) [inline src/components/browse/sidebar/sidebar.tsx:9]: Lets download this css into the assets and use it from there. It freezes the version and also in future if this is not available from this url, we won't miss this css

File: src/components/browse/dataset/datasetList.tsx [hunk 2]
```diff
@@ -1,43 +1,53 @@
...
-        props.dsList === null ?
-          null :
-          (
-            <Dataset dsList={props.dsList} />
-          )
-      }
+        filterDict={props.filterDict}
+      />
+      {props.dsList === null ? null : (
+        <div style={{ height: "60vh"}} className="mb-2 position-relative w-100">
```
Comment (alamgirmani) [inline src/components/browse/dataset/datasetList.tsx:35]: will the hardcoded size have an impact depending on the screen size?

File: src/components/browse/sidebar/filter.tsx [hunk 3]
```diff
@@ -43,15 +43,17 @@ const Filter = (props: filterProps) => {
               <Form.Label
```
Comment (alamgirmani) [inline src/components/browse/sidebar/filter.tsx:43]: The checkboxes are no longer working. It seems to be disabled

File: src/components/browse/sidebar/filter.tsx [hunk 4]
```diff
@@ -43,15 +43,17 @@ const Filter = (props: filterProps) => {
...
                 htmlFor={key}>
+                <Row>
                 <Col xs md lg={10}>
-                  <p className='px-2 my-0'>{option.option}</p>
+                  <p className='ps-2 my-0'>{option.option}</p>
                 </Col>
                 <Col xs md lg={2} className="h-100">
                   <Form.Label className="p-0 m-0"
                     htmlFor={key}>
                     {option.count}
```
Comment (alamgirmani) [inline src/components/browse/sidebar/filter.tsx:53]: Maybe the react-bootstrap badges can be reintroduced here

File: src/components/browse/sidebar/sidebar.tsx [hunk 5]
```diff
@@ -5,43 +5,56 @@ import { Row, Col, Button } from "react-bootstrap";
...
           setSearchResp={props.setSearchResp}
-          limit={props.limit} />
+          limit={props.limit}
+        />
       </Row>
       {props.facetList === null || props.facetList.length < 1 ? null : (
-        <div className="bg-light border p-2 rounded-3">
-          <Row>
+        <div className="bg-light border p-2 rounded-3 pt-3">
+          <Row style={{height: "65vh"}} className="position-relative w-100 px-0 mx-0">
```
Comment (alamgirmani) [inline src/components/browse/sidebar/sidebar.tsx:56]: will the hardcoded height work as intended on various screens of different size?

File: src/components/browse/dataset/datasetList.tsx (hunk 2 above)
Comment (alamgirmani) [reply src/components/browse/dataset/datasetList.tsx:35]: I don't think we should introduce a known bug. Either we introduce the scrollbars after the research or we don't need it now. This is double the effort for no reason.

Comment (ac-jorellanaf) [reply src/components/browse/dataset/datasetList.tsx:35]: It's not a bug, just a missing feature. Having slightly mismatched divs (a few pixels) for a bit should be ok.

File: src/components/browse/sidebar/sidebar.tsx (hunk 1 above)
Comment (ac-jorellanaf) [reply src/components/browse/sidebar/sidebar.tsx:9]: But it's not a CDN. When we install the libraries they are local in the container. We just use the files we downloaded from there. This is done before any calls to the back-end and not done with fron-facing users

File: src/components/browse/dataset/datasetList.tsx (hunk 2 above)
Comment (alamgirmani) [reply src/components/browse/dataset/datasetList.tsx:35]: No, please take your time and provide the right solution. We can't check in hardcoded solutions.

File: src/components/browse/sidebar/sidebar.tsx (hunk 1 above)
Comment (alamgirmani) [reply src/components/browse/sidebar/sidebar.tsx:9]: Ok, lets go with this solution.

File: src/components/browse/sidebar/filter.tsx (hunk 4 above)
Comment (alamgirmani) [reply src/components/browse/sidebar/filter.tsx]: Ok, lets go with this solution and bring it up to the design team.

**Review by alamgirmani — APPROVED** [review]

**Pull request merged**